pkgs/os-specific/linux/kernel/linux-zen.nix

```diff
@@ -12,12 +12,12 @@ buildLinux (args // {
     owner = "zen-kernel";
     repo = "zen-kernel";
     rev = "v${version}-zen1";
```

Comment:
I think we need -zen2 for fsync support. Currently compiling...

Reply:
Yup, it works on -zen2

Reply:
Done 👍